internal/cmd/policy/listrules/listrules.go

```diff
@@ -4,69 +4,34 @@ package listrules
 
 import (
 	"fmt"
-	"strings"
 
+	"github.com/gittuf/gittuf/internal/policy"
 	"github.com/gittuf/gittuf/internal/repository"
 	"github.com/spf13/cobra"
 )
 
-type options struct {
-	targetRef string
-}
+type options struct{}
 
-func (o *options) AddFlags(cmd *cobra.Command) {
-	cmd.Flags().StringVar(
-		&o.targetRef,
-		"target-ref",
-		"policy",
-		"specify which policy ref should be inspected",
-	)
-}
+func (o *options) AddFlags(_ *cobra.Command) {}
 
 func (o *options) Run(cmd *cobra.Command, _ []string) error {
 	repo, err := repository.LoadRepository()
 	if err != nil {
 		return err
 	}
 
-	rules, err := repo.ListRules(cmd.Context(), o.targetRef)
+	policyRules, policyRoot, err := repo.ListRules(cmd.Context(), policy.PolicyRef)
 	if err != nil {
 		return err
 	}
 
-	// Iterate through the rules, they are already in order, and the depth tells us how to indent.
-	// The order is a pre-order traversal of the delegation tree, so that the parent is always before the children.
-
-	for _, curRule := range rules {
-		fmt.Printf(strings.Repeat("    ", curRule.Depth)+"Rule %s:\n", curRule.Delegation.Name)
-		gitpaths, filepaths := []string{}, []string{}
-		for _, path := range curRule.Delegation.Paths {
-			if strings.HasPrefix(path, "git:") {
-				gitpaths = append(gitpaths, path)
-			} else {
-				filepaths = append(filepaths, path)
-			}
-		}
-		if len(filepaths) > 0 {
-			fmt.Println(strings.Repeat("    ", curRule.Depth+1) + "Paths affected:")
-			for _, v := range filepaths {
-				fmt.Printf(strings.Repeat("    ", curRule.Depth+2)+"%s\n", v)
-			}
-		}
-		if len(gitpaths) > 0 {
-			fmt.Println(strings.Repeat("    ", curRule.Depth+1) + "Refs affected:")
-			for _, v := range gitpaths {
-				fmt.Printf(strings.Repeat("    ", curRule.Depth+2)+"%s\n", v)
-			}
-		}
+	policyStagingRules, policyStagingRoot, err := repo.ListRules(cmd.Context(), policy.PolicyStagingRef)
+	if err != nil {
+		return err
+	}
 
-		fmt.Println(strings.Repeat("    ", curRule.Depth+1) + "Authorized keys:")
-		for _, key := range curRule.Delegation.Role.KeyIDs {
-			fmt.Printf(strings.Repeat("    ", curRule.Depth+2)+"%s\n", key)
-		}
+	fmt.Print(policy.GetDiffBetweenPolicyAndStaging(policyRules, policyStagingRules, policyRoot, policyStagingRoot))
```

Comment:
I was also thinking the same thing. I don't think it really matters which PR lands first, but #282 definitely needs some attention, since without it, it is quite hard to know what a mantainer is approving when they create a reference authorization.